### ISSUE TITLE
Version 0.4.6 breaks compiling

### ISSUE THREAD

**Issue by Rindula** [gha-utilities/sass-build#131]
**Describe the bug**

After updating to the new version, the step which uses this action fails with this error:

```
internal/modules/cjs/loader.js:800
    throw err;
    ^

Error: Cannot find module 'sass'

and so on with the stack trace of the module not present...
```

**Repo with error**
[Failed run](https://github.com/Rindula/hirsch/runs/3606299327?check_suite_focus=true)

[Workflow yaml](https://github.com/Rindula/hirsch/blob/7e4e05f92dd8c19b32a208c7d1319b664b98bdff/.github/workflows/ci.yml#L127)

**Comment by S0AndS0** [comment]
Thanks for the detailed report!

My initial suspicion is these errors are totally my fault, and recent updates _should_ fix things.

Please try updating the Workflow file to use version `v0.4.7` and re-executing.

```diff
       - name: Compile CSS from SCSS files
-        uses: gha-utilities/sass-build@v0.4.6
+        uses: gha-utilities/sass-build@v0.4.7
         with:
```

---

For completeness the following is where this bug was likely introduced.

```bash
git whatchanged b8ee599d2fba5a8fb7ee34230ef3661a0c2fd55e -n 1
```


>     Author: S0AndS0 <__email__>
>     Date:   Wed Sep 1 14:43:33 2021 -0700
>
>         :fire: Stop tracking dependency `package.json`
>
>     :100644 000000 25ffb15 0000000 D        node_modules/sass/package.json

**Comment by Rindula** [comment]
Thanks for the fix, v0.4.7 is working 👍
